app/views/content/funding-and-support/tuition-fee-and-maintenance-loans.md

```diff
@@ -18,6 +18,15 @@ cta_arrow_link:
   bursaries:
     link_target: "/funding-and-support/scholarships-and-bursaries"
     link_text: "Find out if you're eligible for a bursary or scholarship"
+  disability-support:
+    link_target: "/funding-and-support/if-youre-disabled"
+    link_text: "Find out more about financial support if you have a disability"
+  parents-carers-support:
+    link_target: "/funding-and-support/if-youre-a-parent-or-carer"
+    link_text: "Find out more about financial support if you’re a parent or carer"
+  veterans-support:
+    link_target: "/funding-and-support/if-youre-a-veteran"
+    link_text: "Find out more about financial support if you’re a veteran"
 navigation: 20.12
 navigation_description: Find out which loans you could get to help pay your fees and living expenses while you train.
 keywords:
```

Comment:
@emmaachesongray Would suggest deleting the keywords on this and all the other files on this PR given you are editing them (and the PR isn't ready yet!)